superset/config.py

```diff
@@ -327,6 +327,8 @@ def _try_json_readsha(  # pylint: disable=unused-argument
     "DISPLAY_MARKDOWN_HTML": True,
     # When True, this escapes HTML (rather than rendering it) in Markdown components
     "ESCAPE_MARKDOWN_HTML": False,
+    "GLOBAL_ASYNC_QUERIES": False,
+    "GLOBAL_ASYNC_QUERIES_OPTIONS": {"transport": "polling", "polling_delay": 250},
```

Comment:
Ah, I really think we should keep feature flags booleans... What I imagined was to add the options as a global config value, add a separate export entry [here](https://github.com/apache/incubator-superset/blob/f2d9f0001302c548216b542867fa0ddf3ed483cc/superset%2Fviews%2Fbase.py#L308), change [initiFeatureFlags](https://github.com/apache/incubator-superset/blob/f292015ccda28191dbde54324ec6b02bc65445fa/superset-frontend%2Fsrc%2FfeatureFlags.ts#L52) to something like `initCommonBootstrap()`, and add such global options there. Or, pass it to Redux via each page's `getInitialState`.

But if you find it easier to expose these in the feature flags, we can always refactor later.

Reply:
You should be able to expose config vars to the frontend by adding them [here](https://github.com/apache/incubator-superset/blob/7e6f04fce4321cee9d69b14fa003e4c24b5cf720/superset/views/base.py#L68), then they should be available in `boostrap_data.common`

Reply:
Yep, agreed this it not ideal, and that Feature Flags should remain booleans. I'll refactor.

Reply:
Updated